CHANGELOG.md

```diff
@@ -5,14 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [2.2.2] - 2023-09-19
```

Comment:
```suggestion
## [Unreleased]
```

Reply:
@bbakernoaa Let's keep it as unreleased. We don't know when the PR will go in and what the release number will be or the like.